articles/reporting/spam-trap-monitoring.md

```diff
@@ -0,0 +1,50 @@
+---
+title: "Spam Trap Monitoring"
+description: "The Spam Trap Monitoring surfaces how much of your mail you are sending to known spam traps. This can help you find problems in your list collection and hygiene practices, and avoid deliverability problems."
+---
+
+The Spam Trap Monitoring surfaces how much of your mail you are sending to known spam traps. This can help you find problems in your list collection and hygiene practices, and avoid deliverability problems.
+
+## Spam traps
+
+Spam traps, also known as honeypots, are email addresses that are set up by blacklists, filter companies, and mailbox providers as a way to identify senders with bad list hygiene and list collection practices.
+
+As a sender, these addresses have no value since no one is reading the email. When you send to a high number of traps, your mail will be blocked or placed in the spam folder. In sum, you should avoid sending to spam traps as much as possible.
+
+[Learn more](https://www.sparkpost.com/email-deliverability-guide/spam-traps/) about spam traps and how to avoid them.
+
+### Monitored traps
+
+SparkPost has visibility into multiple commercial and non-commercial spam trap networks for the broadest coverage in the industry. Every day we analyze all the emails that were sent to to find and monitor new spam traps to provide the most accurate report of your sending possible.
+
+### Trap types
+
+Spam traps are categorized into different types based on their behavior and the way they can end up on your list.
+
+Knowing the type spam trap you’re sending to can help you find where in your email program may have an issue.
+
+* **Recycled** - Email addresses that once were real, but have been converted into a spam trap. If you have these on your list it’s probably a sign you need to clean out old unengaged recipients. You can look at your engagement recency cohorts (EU) to see if you are sending to a lot of unengaged recipients.
```

Comment:
are the links missing?  for "engagement recency cohorts (EU)"?

Reply:
yup, great catch!